tests/test_destination_toctou.py

```
@@ -0,0 +1,999 @@
+        self.assertTrue(os.path.exists(guard))
+        self.assertEqual(Path(guard).read_bytes(), b"existing")
+        self.assertEqual(self.dest.read_bytes(), b"claimed by somebody else")
+        self.assertIn("was not deleted", output.getvalue())
+
+
+class ReadOnlyDestinationDirectoryTests(OperationHarness):
+    def test_a_commit_into_a_read_only_directory_fails_closed(self):
+        os.chmod(self.dest_dir, stat.S_IRUSR | stat.S_IXUSR)
+        self.addCleanup(os.chmod, self.dest_dir, stat.S_IRWXU)
+
+        result, _moved, output = self.run_operation()
```

Ruff (0.16.2)

[warning] Unpacked variable `output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)